src/models.py

```diff
@@ -68,6 +68,6 @@ def forward(self, x_flux_vector, x_state_vector, time_vector):
         x_H_II_prediction = torch.sigmoid(output[:, 0])
         x_He_II_prediction = torch.sigmoid(output[:, 1])
         x_He_III_prediction = torch.sigmoid(output[:, 2])
-        T_prediction = torch.pow(10, output[:, 3])
+        T_prediction = torch.pow(10, 13 * torch.sigmoid(output[:, 3]))
```

Comment:
Please add a comment here on what the idea behind this approach was for later reference.